src/DynamoCore/BuiltInAndOperators/Operators.xml

```diff
@@ -6,106 +6,119 @@
             <summary>
-                Finds the remainder of x/y
+               Modulo: Finds the remainder of x/y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="number">Remainder of the x input after being divided by the y input</returns>
             <search>mod, modulo, remainder</search>
         </member>
 		<member name="%le">
             <summary>
-                x less or equals y?
+              If the x input is less than the y input, or if the two values are equal, it will return true. If x is greater than y, this operator will return false.
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="bool">Result of check</returns>
             <search>less,equal</search>
         </member>
 		<member name="%lt">
             <summary>
-                x less y?
+              If the x input is less than the y input, it will return true. If x is greater than y, or if the two value are equal, this operator will return false.
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="bool">Result of check</returns>
             <search>less</search>
         </member>
 		<member name="%and">
             <summary>
-                x and y?
+                It takes two boolean values as input and returns true if both values are true.
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">A boolean to check if true</param>
+            <param name="y">Another boolean to check if true</param>
+            <returns name="bool">Result of check</returns>
             <search>and</search>
         </member>
 		<member name="%or">
             <summary>
-                x || y?
+              Logical OR operator
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="bool">Boolean to check if true</param>
+            <param name="otherBool">Other boolean to check if true</param>
+            <returns name="bool">True if any of the inputs are true</returns>
             <search>or</search>
         </member>
 		<member name="%nq">
             <summary>
-                x doesn't equal y.
+                It takes two input values and returns true if the two values are not equal to each other and returns false if the two values are equal.
```

Comment:
NIT: Maybe `Returns true if the two values are different` would be a simpler way to put it.

Reply:
Addressed in 0e11365